deploying-a-django-app-on-okteto.md

```diff
@@ -0,0 +1,704 @@
+        permission_classes = [IsAuthenticated, OnlyInvestorOwnerUserPermission]
+        def get(self, request):
+            user = User.objects.get(username=request.user.username)
+            user_serializer = UserSerializer(user)
+            message = "You can only see this if you're an investor or owner"
+            return Response(
+                {"user": user_serializer.data, "message": message, "success": True},
+                status=status.HTTP_200_OK,
+            )
+    
+    class OnlyOwnerUser(GenericViewSet):
+        permission_classes = [IsAuthenticated, OnlyOwnerUserPermission]
+        def get(self, request):
+            user = User.objects.get(username=request.user.username)
+            user_serializer = UserSerializer(user)
+            message = "You can only see this if you're an owner"
+    
+            return Response(
+                {"user": user_serializer.data, "message": message, "success": True},
+                status=status.HTTP_200_OK,
+            )
+
+`OnlyStaffOwnerUser` can only be accessible to authenticated staff and Owner, using the permission classes `IsAuthenticated` and `OnlyStaffOwnerUserPermission`.
+
+`OnlyAdminStaffOwnerUser` is only available to admin staff and Owner. 
+`OnlyInvestorAndOwnerUser` is also only available to Investors and Owner
+`OnlyOwnerUser` is only available to owners.
+These are through the permission classes attached to them. 
+
+Add the following to your `urls.py`:
+Here were are assigning the views we created to routes:
+
+
+    from users.views import *
+    
+    urlpatterns = [
+        path('admin/', admin.site.urls),
+    ]
+    
+    urlpatterns += [
+        path('user-auth/', LoginView.as_view({"post": "login"}), name = "user-login"),
+        path('user-signup/', SignUpView.as_view({"post": "create"}), name = "user-signup"),
+        path('all-user/', AllUser.as_view({"post": "get"}), name = "all-users"),
+        path('authenticated-user/', OnlyAuthenticatedUser.as_view({"post": "get"}), name = "authenticated-user"),
+        path('only-staff/', OnlyStaffOwnerUser.as_view({"post": "get"}), name = "only-staff"),
+        path('only-admin/', OnlyAdminStaffOwnerUser.as_view({"post": "get"}), name = "only-admin"),
+        path('only-investor-owner/', OnlyInvestorAndOwnerUser.as_view({"post": "get"}), name = "only-investor-owner"),
+        path('only-owner/', OnlyOwnerUser.as_view({"post": "get"}), name = "only-owner"),
+    ]
+## Deploying to Okteto
+
+Now we will  be deploying our API and to Docker and Okteto Cloud.
+
+Get an account on Okteto and run this command;
+
+    $ okteto namespace
+
+Next we try building and deploying to Okteto:
+
+
+    $ okteto stack deploy --build
+
+This creates our Docker image from our `Dockerfile`, and builds the services in the `okteto-stack.yml` file. 
+
+Next, go to the Okteto Dashboard settings, and the secrets tab and add your secret key(gotten from your django settings or .env) for your Docker image:
```

Comment:
This step doesn't work, since the secrets are only available during the deployment of a pipeline. (https://okteto.com/docs/cloud/secrets). I think for your post it's better to ignore this step, and instead use the `env_file` property of Okteto Stacks (https://okteto.com/docs/reference/stacks#env_file-string-optional).